core-java-modules/core-java-lang-operators-2/src/test/java/com/baeldung/integersatbitlevel/IntegersBitLevelUnitTest.java

```diff
@@ -0,0 +1,25 @@
+package com.baeldung.integersatbitlevel;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class IntegersBitLevelUnitTest {
+    @Test
+    public void givenKeyAndPlaintext_whenEncryptingPlaintext_thenRetrieveCiphertext() {
+        int key = 0b10101010;
+        int plaintext = 0b11001100;
+        int ciphertext = plaintext ^ key;
+
+        assertEquals(ciphertext, 0b01100110);
+    }
+
+    @Test
+    public void givenDataANDMask_whenApplyingMask_thenRetrieveIsolatedBits() {

```

Comment:
Here we mean `AND` as logical AND, where previously we've used `and` to mean _also_. I think it's a little confusing.

How about `givenData_whenApplyingMaskWithBitwiseAnd_thenLeastSignificantBitsReturned`